### PR TITLE
Ficus: improve accessibility

### PR DESCRIPTION
**Pull request by amirtds**
Improve accessibility based on https://github.com/edx/edx-platform/pull/19944 and https://github.com/edx/edx-platform/pull/19942

### CODE REVIEW

**Comment by amirtds** [comment]
[(2) PSU Ginkgo staging: Merge 2 PRs that fix 2 more accessibility issues on video player](https://trello.com/c/RKgZWZQm/3665-2-psu-ginkgo-staging-merge-2-prs-that-fix-2-more-accessibility-issues-on-video-player)

File: common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js [hunk 1]
```diff
@@ -136,7 +136,7 @@
             fullScreenClassNameEl.removeClass('video-fullscreen');
             $(window).scrollTop(this.scrollPos);
             this.videoFullScreen.fullScreenEl
-            .attr('title', gettext('Fill browser'))
+            .attr({title: gettext('Fill browser'), 'aria-label': gettext('Fill browser')})
```
Comment (bryanlandia) [inline common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js]: shouldn't `title` be quoted here?

File: common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js [hunk 2]
```diff
@@ -158,7 +158,7 @@
             this.videoFullScreen.fullScreenState = this.isFullScreen = true;
             fullScreenClassNameEl.addClass('video-fullscreen');
             this.videoFullScreen.fullScreenEl
-            .attr('title', gettext('Exit full browser'))
+            .attr({title: gettext('Exit full browser'), 'aria-label': gettext('Exit full browser')})
```
Comment (bryanlandia) [inline common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js]: and here

File: common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js [hunk 3]
```diff
@@ -74,7 +74,7 @@
                 this.el
                 .addClass('pause')
                 .removeClass('play')
-                .attr('title', gettext('Pause'))
+                .attr({title: gettext('Pause'), 'aria-label': gettext('Pause')})
```
Comment (bryanlandia) [inline common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js]: and here

File: common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js [hunk 4]
```diff
@@ -84,7 +84,7 @@
                 this.el
                 .removeClass('pause')
                 .addClass('play')
-                .attr('title', gettext('Play'))
+                .attr({title: gettext('Play'), 'aria-label': gettext('Play')})
```
Comment (bryanlandia) [inline common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js]: and here

**Pull request merged**